client/src/components/BrowseChefs/Chefs.js

```diff
@@ -52,11 +42,24 @@ const Chefs = ({ chefProfileUrl, name, location, cuisineTags, bio }) => {
               <Typography variant="body2">{location}</Typography>
             </Box>
             <Box mt={3} mb={3}>
-              <Chip
-                label={mapCuisineTags(cuisineTags)}
-                color="secondary"
-                className={classes.cuisineChip}
-              />
+              {cuisineTags.length > 0 ? (
+                cuisineTags.map((cuisineTag, i) => {
+                  return (
+                    <Chip
+                      key={`${name}${i}`}
```

Comment:
It's generally not good practice to use the index as a key in React, because if the index changes it will re-render the component. In this case I don't think that's a problem because we wouldn't expect it to change, but good to know for the future